docs/appendix/apis/components/input-controller.md

```diff
@@ -0,0 +1,573 @@
+---
+title: "Input controller API"
+linkTitle: "Input Controller"
+titleMustBeLong: true
+weight: 20
+type: "docs"
+description: "The input controller API allows you to give commands to your input controller components with code instead of with the graphical interface of the Viam app"
```

Comment:
```suggestion
description: "Commands for configuring callbacks for events, allowing you to configure input devices to control your machines."
```
This "configure" word is confusing though (pulled from line 12)--how does this relate to what we usually call the config?